src/Scoreboard.java

```diff
@@ -0,0 +1,40 @@
+import java.util.ArrayList;
+
+public class Scoreboard {
+
+    private ArrayList<Ball> score = new ArrayList<>();
+    private Ball bonusBall;
+
+    public Scoreboard() {
+    }
+
+    protected void placeBall(Ball ball){
+        score.add(ball);
+    }
+
+    protected void placeBonusBall(Ball ball){
+        bonusBall = ball;
```

Comment:
Je zou hier nog een validatiestap toe kunnen voegen, dat alleen een `bonusBall` kan worden geplaced als er nog geen `bonusBall` is, e.g.:

```
  if (bonusBall != null) {
    throw new IllegalArgumentException("bonus ball already placed");
  }
```

Reply:
Will do! Cheers